index.js

```diff
@@ -158,11 +159,46 @@ export async function createSpace (name) {
   console.log(space.did())
 }
 
+function findAccountsThatCanProviderAdd (client) {
+  const accounts = []
+  const proofs = client.proofs()
+  for (const proof of proofs) {
+    const allows = ucanto.Delegation.allows(proof)
+    for (const resourceDID of Object.keys(allows)) {
+      if (resourceDID.startsWith('did:mailto:') && allows[resourceDID]['provider/add']) {
+        accounts.push(resourceDID)
+      }
+    }
+  }
+  return accounts
+}
+
+// TODO: extract to external library along with its counterpart in https://github.com/web3-storage/w3protocol/blob/main/packages/access-client/src/utils/did-mailto.js
```

Comment:
good idea, inverse of https://github.com/web3-storage/w3protocol/blob/main/packages/access-client/src/agent.js#L33

Reply:
https://github.com/web3-storage/w3protocol/issues/601